internal/config/moduleconfig/module_config.go

```diff
@@ -158,3 +178,38 @@ func findMissing(obj reflect.Value, path, metadata string, missing *[]string) {
 		}
 	}
 }
+
+// SummarizeParameters receives all parameters gathered from prompts during `Zero init`
+// and based on module definition to construct the parameters of interest for each module
+func SummarizeParameters(module ModuleConfig, allParams map[string]string) map[string]string {
+	moduleParams := make(projectconfig.Parameters)
+	// Loop through all the prompted values and find the ones relevant to this module
+	for parameterKey, parameterValue := range allParams {
+		for _, moduleParameter := range module.Parameters {
+			if moduleParameter.Field == parameterKey {
+				if moduleParameter.OmitFromProjectFile {
+					flog.Debugf("Omitted %s from %s", parameterKey, module.Name)
+				} else {
+					moduleParams[parameterKey] = parameterValue
+				}
+			}
+		}
+	}
+	return moduleParams
+}
+
+// SummarizeConditions collects conditions from module-definition
+// then append the condition into the zero-project configuration
+func SummarizeConditions(module ModuleConfig) []projectconfig.Condition {
+	moduleConditions := []projectconfig.Condition{}
```

Comment:
Not a big deal in this case and probably not worth a change, but in general if you know the final size of the slice ahead of time like this it's more efficient to pre-allocate it. For example, 
```
moduleConditions := make([]projectconfig.Condition{}, len(module.Conditions))
```
then as you loop through just assign each value like `moduleConditions[i] = newCond`